backend/src/main/java/ee/tlu/forum/model/Post.java

```diff
@@ -0,0 +1,31 @@
+package ee.tlu.forum.model;
+
+import com.sun.istack.NotNull;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+public class Post extends BaseEntity {
+
+    @Id
+    private Long id;
+
+    @NotNull
+    private String text;
+
+    @ManyToOne(fetch = FetchType.LAZY, optional = false)
+    @JoinColumn(name = "user_id", nullable = false)
+    private User author;
+
+    @ManyToOne(fetch = FetchType.LAZY, optional = false)
+    @JoinColumn(name = "thread_id", nullable = false)
+    private Thread thread;
```

Comment:
Shouldn't you be importing Thread in order to use this?

Reply:
```
package ee.tlu.forum.model;
```
It is because both are in same package (folder). They are automatically imported - no need to write an import statement for that. The more you know! :)

Reply:
Aha! That's cool. And really good to know :D